Competition/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -42,6 +45,10 @@ public void setupDriveCommands(
         pursuit.addPoint(new RabbitPoint(3*12, 3*12, 0));
         operatorInterface.gamepad.getifAvailable(2).whenPressed(pursuit);
         
+        goFromHabTouchingRampToCargoShipFrontRight.setMode(PointLoadingMode.Absolute);
+        goFromHabTouchingRampToCargoShipFrontRight.addPoint(poseSubsystem.cargoShipFrontRightManager.getPose());
```

Comment:
Instead of accessing the point manager directly (and it needing to be public), call `poseSubsystem.getFieldPoseForLandmark()`